lib/extend/tag.js

```diff
@@ -214,12 +214,7 @@ class Tag {
     this.env.addExtension(name, tag);
   }
 
-  render(str, options, callback) {
-    if (!callback && typeof options === 'function') {
-      callback = options;
-      options = {};
-    }
-
+  render(str, options = {}) {
```

Comment:
`Tag.render()` just doesn't have callback, so I remove those lines and `callback` parameter directly.

Reply:
Seems like a callback is promisified?

https://github.com/hexojs/hexo/blob/640b91e4b13d37a9fbdaf518a296354345bdf7ff/lib/extend/tag.js#L229